cmd/operator/main.go

```diff
@@ -39,6 +82,10 @@ func main() {
 	flag.StringVar(&namespace, "namespace", "default", "The namespace in which the operator runs")
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&healthProbeAddr, "health-probe-bind-address", ":8081", "The address the health probe endpoint binds to.")
+	images := kvarg{}
+	flag.Var(&images, "images", "Images to use for containers managed by the observability-operator.")
+	versions := kvarg{}
+	flag.Var(&versions, "versions", "Version of containers managed by the observability-operator.")
```

Comment:
Do we need to define the operand versions or should we rely on the hardcoded default versions from github.com/rhobs/obo-prometheus-operator?

Reply:
I was going to argue for the p-o default versions. In the Thanos Querier controller I already set that explicitly https://github.com/rhobs/observability-operator/pull/386/files#diff-62df723ef5f27edbbfaa7feeb8a0612efaa9ef8d91828004287a17f77f0c48aeR74. In `Prometheus` objects the version is not set if empty to let p-o do its defaulting.

Reply:
Defaulting would be latest or does p-o do any kind of logic already?

Reply:
IMHO it makes more sense for p-o and thanos do their defaults